src/components/Models/EnvironmentModel/EnvironmentModel.component.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from 'react';
+import { useFallback, useGltfLoader, CustomNode, Transform } from 'src/services';
+import { useGraph } from '@react-three/fiber';
+import { BaseModelProps } from 'src/types';
+import { EnvironmentModels } from 'src/services/Environment.service';
+
+export interface EnvironmentModelProps extends BaseModelProps {
+  environment: string | EnvironmentModels;
+  scale?: number;
+}
+
+export const EnvironmentModel: FC<EnvironmentModelProps> = ({ environment, scale = 1, setModelFallback }) => {
```

Comment:
let's add onLoaded that will trigger background props

Reply:
done